plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -651,6 +651,7 @@ function BookInfoManager:cleanUp()
 end
 
 local function findFilesInDir(path, recursive)
+    local stringStartsWith = require("util").stringStartsWith
```

Comment:
Oh, yeah, good idea, makes the whole thing slightly less clunky to read ;).

Reply:
Not really intended for that :) I just thought that if I used `local util = require("util")` here, luacheck would complain about it shadowing the top one (and I didn't want to name it `frontutil` :)